pkgs/applications/graphics/goxel/default.nix

```diff
@@ -3,28 +3,29 @@
 
 stdenv.mkDerivation rec {
   pname = "goxel";
-  version = "0.10.8";
+  version = "0.11.0";
 
   src = fetchFromGitHub {
     owner = "guillaumechereau";
     repo = "goxel";
     rev = "v${version}";
-    sha256 = "sha256-M9H9SV8xmU7Jw5rEdV0gfloIEBvWmWSuH+BCrowpf2M=";
+    sha256 = "sha256-FrPoKJXzWzfUF0ITYxIUdHi0kCh2VJ+A093HaXxMofA=";
   };
 
   patches = [ ./disable-imgui_ini.patch ];
 
   nativeBuildInputs = [ scons pkg-config wrapGAppsHook ];
   buildInputs = [ glfw3 gtk3 libpng12 ];
-  NIX_LDFLAGS = "-lpthread";
 
   buildPhase = ''
+    runHook preBuild
     make release
+    runHook postBuild
   '';
```

Comment:
```suggestion
  makeTarget = [ "release" ];
```

Reply:
Duh, I forgot about that. fixed.

Reply:
Nevermind, this will probably trigger scons' build phase. and there's only `dontUseSconsInstall`, so it can't be overridden that way

Reply:
maybe I could add a `dontUseSconsBuild` check to the scons hook too. what do you think?

Reply:
> maybe I could add a `dontUseSconsBuild` check to the scons hook too. what do you think?

Yeah, we probably should.



> ugh, but that would be a mass rebuild

then we could do an extra PR and send that to staging.